README.md

```diff
@@ -0,0 +1,15 @@
+**Note:** This extension was part of the TYPO3 Core until TYPO3 v10 and is 
```

Comment:
`until TYPO3 v10` → shouldn't that be `prior to TYPO3 v10`? Just to avoid confusions...

Reply:
This was the claim in our TYPO3-Documentation repos that we deprecated and archived, but we can adapt it if you want.

Reply:
Hm, personally I'm fine with that... if it's like that in all documentation, and users did understand it so far, there's no need to change it right now.